choose/command.go

```diff
@@ -33,7 +33,17 @@ func (o Options) Run() error {
 	}
 
 	theme := huh.ThemeCharm()
-	options := huh.NewOptions(o.Options...)
+	options := make([]huh.Option[string], len(o.Options))
+	for i, option := range o.Options {
+		delimIndex := strings.Index(option, o.Deliminator)
```

Comment:
Can we use `strings.Split` here (just a personal preference)

Reply:
Sometimes there may be two ":" in the string or no ":" in the string. I think it is easier to determine these situations using the `strings.Index`.

Reply:
I have now switched to using `strings.SplitN`, maybe it looks better!